src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -67,6 +70,14 @@ export class SearchResults extends Component {
     };
   }
 
+  componentDidUpdate(previousProps) {
+    const justRefreshed = previousProps.fetching && !this.props.fetching;
+
+    if (justRefreshed) {
+      focusElement('[data-forms-focus]');
+    }
+  }
```

Comment:
After results are done loading, we focus on the `Displaying 1 - 10 out of 48 results for "health care"` message or `No results were found...`

Reply:
VO + Chrome didn't announce the dash, but I'm not going to worry about it until we test in a proper browser + screen reader pairing. It might be necessary to add a `<span aria-hidden="true">-<span>` and another SR-only to make screen readers read the string properly, but we'll cross that bridge when we get there.

Reply:
Interesting. I just pushed a commit to switch the default dash to an "en dash", which is the formal character to represent a range of numbers. Kind of an interesting article - https://en.wikipedia.org/wiki/Dash#Figure_dash - plus all the HTML entities representing dashes - https://www.w3schools.com/charsets/ref_utf_punctuation.asp

Reply:
I'm curious if the explicit en dash will fix that

Reply:
In my experience the screen readers don't differentiate between the various dashes, sadly. :cry: